tinyagentos/channel_hub/matrix_connector.py

```
@@ -0,0 +1,127 @@
+            try:
+                await self._client.close()
+            except Exception as exc:
+                logger.debug("Matrix client close error: %s", exc)
+            self._client = None
+
+    async def _sync_loop(self):
+        try:
+            await self._client.sync_forever(timeout=30000, full_state=False)
+        except asyncio.CancelledError:
+            pass
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)